src/views/product-tutorials-view/server.ts

```diff
@@ -30,7 +39,7 @@ export async function getProductTutorialsPageProps(
   return {
     props: stripUndefinedProperties({
       collections: filteredCollections,
-      product,
+      product: { ...product, ...productData },
```

Comment:
So far, the duplicate properties I'm seeing are `name` and `slug`, which need to be defined higher up than the Learn content so other places in the app can use them. Here are my thoughts on the data coming from the Learn API:

- `docsUrl` not sure how this is used in Learn yet, but I assume we'll be able to programmatically determine this in DevDot and possibly add it to `ProductData` (or maybe something call `LearnProductData`?)
- `id` can possibly be renamed `learnId` since it's only used in the scope of Learn content
- `description` can also possibly renamed `learnDescription` so that it's scoped like `id`